ee/clickhouse/models/property.py

```diff
@@ -163,6 +164,7 @@ def parse_prop_clauses(
             else:
 
                 if person_properties_mode == PersonPropertiesMode.EXCLUDE:
+                    # TODO: I don't get why this is here??
```

Comment:
I made it more clear by changing the condition to be == `USING_SUBQUERY` so that means the cohort filter will use the costly subquery only if explicitly not joining in person tables, otherwise use the more optimal logic